android/java/org/chromium/chrome/browser/settings/NetworkPreferenceAdapter.java

```diff
@@ -0,0 +1,544 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
I kept the original year in the copyright header as this class has been renamed from `NetworkPreferenceAdapter.java`.